### DISCUSSION TITLE
Hello  my name is Andrew and i would like to talk to you about..

### DISCUSSION THREAD

**Discussion by oklahomaguy23** [mnikonov/gpt-labs#44] [Ideas]
Hello, my name is Andrew. and I am currently in the process of using your app to work on a chatbot for veterans with PTSD. I am hoping that I can integrate this software into a Windows version of that.  With that all being said I am a Veteran With PTSD and TBI.  Before I deployed overseas I earned an associate's degree in software development. Due to my TBI, it makes it very hard to remember to spell let alone code. It revived a spark in me that I had lost.  I look forward to hearing from everyone on this idea.

**Comment by Freeking2510** [comment]
سلام
